sphinx/conf.py

```diff
@@ -118,6 +118,7 @@
     ("py:attr", "django.contrib.auth.models.User.feedback"),
     ("py:attr", "django.contrib.auth.models.User.mfa_keys"),
     ("py:attr", "django.contrib.auth.models.User.imprint_translations"),
+    ("py:attr", "django.utils.datastructures.MultiValueDict"),
```

Comment:
```suggestion
    ("py:class", "django.utils.datastructures.MultiValueDict"),
```
I think this is a class and not an attribute.